blazor/treeview/check-box.md

```diff
@@ -99,4 +101,646 @@ By default, the checkbox state of parent and child nodes are dependent on each o
 }
 ```
 
-![Blazor TreeView with CheckBox](./images/blazor-treeview-checkbox.png)
\ No newline at end of file
+![Blazor TreeView with CheckBox](./images/blazor-treeview-checkbox.png)
+
+## Check nodes through API
+
+The Blazor TreeView component enables the ability to check specific nodes upon initial rendering through the use of the [CheckedNodes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTreeView-1.html#Syncfusion_Blazor_Navigations_SfTreeView_1_CheckedNodes) property. This property allows for the checkbox selection of nodes by passing in an array collection of node IDs as strings.
+
+In the following example, the `CheckedNodes` property is used.
```

Comment:
The line is not needed. (across all UGs).

Reply:
Done.